.github/actions/install-debian-packages/action.yml

```diff
@@ -19,10 +19,13 @@ runs:
     - name: Install debian packages
       shell: bash
       run: |
+        # There is no way to clear the cache and llvm-strip is needed by bpf2go.
+        # For now, we will add it here.
+        sudo apt install llvm
```

Comment:
What about updating the `key` used in `actions/cache@v2`? 

Btw, now that I look with more detail at the logic to install the deb packages, isn't it possible to move all usages of `actions/cache@v2` from `workflows/inspektor-gadget.yml` to `install-debian-packages/action.yml` to avoid repeating that so many times?